spring-rabbit/src/main/resources/META-INF/spring/aot.factories

```diff
@@ -0,0 +1 @@
+org.springframework.amqp.rabbit.aot.RabbitRuntimeHintsRegistrar=org.springframework.amqp.rabbit.aot.RabbitRuntimeHintsRegistrar
```

Comment:
The key in this file is wrong. Must be: `org.springframework.aot.hint.RuntimeHintsRegistrar`

Reply:
Yes; I fixed that last night (but failed to re-push).